docs/tour/why_wasmcloud.mdx

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 3
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Hello Universe
+
+You've now built a simple microservice with wasmCloud that has common capabilities (handle HTTP requests, interface with a Key-Value store) that many cloud native applications need. So, why wasmCloud? What makes this application that you built different from similar apps on other platforms? Beyond providing an easy-to-use platform that's quick to get started with, we also want to make sure that you understand that the biggest benefits of wasmCloud come after you've built your application.
+
+Consider, now that you have your little microservice, what you may have to do to actually deploy that to production in the cloud. If you're building this application in a modern language and packaging it up into a container, you may have the following hurdles to consider _before you even get to the point of writing your features in the first place_:
+
+1. What CPU architecture am I going to be deploying to? Do I need to build for x86, ARM, or both?
+2. What operating system am I going to be deploying to? Do I need to support more than just Linux?
+3. What cloud am I going to deploy to? What if I need to deploy to multiple clouds?
+4. Do I need to be able to vertically scale my application?
+5. Do I need to be able to horizontally scale across multiple instances of my application? What about high availability? How will I handle communication between instances of my application?
+
+**All of the above questions should not influence your application's functional logic, don't need to be considered until after you've already built your application with wasmCloud, and can be freely changed at runtime when you need to.**
```

Comment:
I would lead with this callout before the list, and maybe style the list with strikethrough

Reply:
Curious if you like the changes, moved up and into an `info` block with a little bit of rewording.